apps/01PigLatin.js

```diff
@@ -7,11 +7,128 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:
+  //
+  // String.prototype.toLowerCase is a method that returns the value of a called string into lowercase letters. One of the tests needed to be passed requires that this app should auto lowercase the word before translation.
+  //
+  //
+  // By using .toLowerCase here we ensure that the word will be entirely lowercase before any other changes are made. We also limit the number of times we have to search for each vowel.
+  //
+  //
+  // Since 'a' !== 'A' we would normally need to search for the uppercase and lowercase strings.
+  //
+  // Since .toLowerCase transforms every letter in a string into lowercase we only need to search for each vowel one time.
+  //
+  //
+  word  = word.toLowerCase();
 
+  // Concept Check
+  //
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer:
+  //
+  // An array searc that returns a value of -1 indicates that the value being searched for cannot be found.
```

Comment:
True. You can do similar things with arrays, but here we are working with strings.